nixos/modules/config/i18n.nix

```
@@ -132,10 +132,14 @@
           )) != [ ]
         )
         ''
-          It looks like that you've overriden `i18n.supportedLocales`, but
-          forgot to include some locales specified in `i18n.defaultLocale`,
-          `i18n.extraLocales` or `i18n.extraLocaleSettings`. If you're trying to
-          install additional locales, use `i18n.extraLocales` instead.
+          `i18n.supportedLocales` is deprecated in favor of `i18n.extraLocales`,
+          and it seems you are using `i18n.supportedLocales` and forgot to
+          include some locales specified in `i18n.defaultLocale`,
+          `i18n.extraLocales` or `i18n.extraLocaleSettings`.
+
+          If you're trying to install additional locales not specified in use
```

Did you use exactly what I wrote :smile: ?

```suggestion
          If you're trying to install additional locales not specified in
```